Frends.SFTP.WriteFile/Frends.SFTP.WriteFile.Tests/Lib/Helpers.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Security.Cryptography;
+using Renci.SshNet;
+using Renci.SshNet.Common;
+using Frends.SFTP.WriteFile.Definitions;
+using Frends.SFTP.WriteFile.Enums;
+
+namespace Frends.SFTP.WriteFile.Tests;
+
+internal static class Helpers
+{
+    /// <summary>
+    /// Test credentials for docker server.
+    /// </summary>
+    private static string _dockerAddress = "localhost";
+    private static string _dockerUsername = "foo";
+    private static string _dockerPassword = "pass";
```

Comment:
All of these properties can be `readonly`.

Reply:
Fixed.